### PR TITLE
CM-352 added set event_id for missing event types

### PR DESCRIPTION
**Pull request by bmxpadu**
### Jira ticket:
https://berlingskemedia.atlassian.net/browse/CM-352

### Pre/Post release actions:
(none)

### CODE REVIEW

File: server/notifications_events.js [hunk 1]
```diff
@@ -274,3 +254,31 @@ function slimDownAllAcctPlansM(all_acct_plans_m) {
...
+      && event.eventPayload.request.transaction_id
+  ) {
+    return event.eventPayload.request.transaction_id
+  }
+  if (event.eventIdent && event.eventIdent.event_guid) {
+    return event.eventIdent.event_guid;
+  }
+  if (event.userAuditLogRequestDetails) {
+    const { requestActionType, requestID } = event.userAuditLogRequestDetails;
+    return `${requestActionType}_${requestID.replace(/^(req_id_\\s_)|^(req_id_)/, '')}`;
```
Comment (bmxsele) [inline server/notifications_events.js:273]: Can't we just do: 

```suggestion
    return requestID.replace('req_id', requestActionType);
```

?

**Review by bmxsele — CHANGES_REQUESTED** [review]

File: server/notifications_events.js (hunk 1 above)
Comment (bmxsele) [reply server/notifications_events.js:273]: Or:

```suggestion
    return `${requestActionType}_${requestID.replace('req_id_', ''}.trim()`;
```

Comment (bmxsele) [reply server/notifications_events.js:273]: OK, we can't - discussed on slack

Comment (dgolabek-sii) [reply server/notifications_events.js:273]: `return requestID.replace(/^req_id_(\\s_)?/, requestActionType);`

File: server/notifications_events.js [hunk 2]
```diff
@@ -274,3 +254,31 @@ function slimDownAllAcctPlansM(all_acct_plans_m) {
...
+  }
+  if (event.eventPayload
+      && event.eventPayload.request
+      && event.eventPayload.request.transaction_id
+  ) {
+    return event.eventPayload.request.transaction_id
+  }
+  if (event.eventIdent && event.eventIdent.event_guid) {
+    return event.eventIdent.event_guid;
+  }
```
Comment (dgolabek-sii) [inline server/notifications_events.js:270]: as I understand we get events in the format corresponding to this code and not like in the ticket description - `AMPSEventGUID` ?

Comment (bmxpadu) [reply server/notifications_events.js:270]: added it

**Review by dgolabek-sii — APPROVED** [review]

**Review by bmxsele — APPROVED** [review]

**Review by bmxsele — APPROVED** [review]

**Pull request merged**